src/content/docs/reference/.gitignore

```
@@ -1,5 +1,4 @@
 config.md
-acl/*.md
 cli.mdx
```

```suggestion
cli.mdx
acl/*.md
!acl/core-permissions.mdx
```

idk if the "un-ignore" line is really needed with the different file extensions but the auto-generated .md files should still be ignored.

---

I think this change wasn't needed, you're right